bookkeeper-server/src/main/java/org/apache/bookkeeper/server/EmbeddedServer.java

```diff
@@ -0,0 +1,630 @@
+/*
+ * Copyright 2022 The Apache Software Foundation.
```

Comment:
This license seems wrong, please copy/paste from other files

Reply:
Changed with a copy took from other files